tests/test_group.py

```diff
@@ -18,7 +19,7 @@ def test_group_description_edit(self, base_url, selenium, vouched_user):
         home_page.login(vouched_user['email'], vouched_user['password'])
 
         # Create a new group
-        group_name = (time.strftime('%x-%X'))
+        group_name = (uuid.uuid4())
```

Comment:
The change looks good, but there are also three other uses of `time.strftime` in this file.